cdap-security/src/main/java/io/cdap/cdap/security/authorization/SecurityMetricsService.java

```diff
@@ -55,7 +56,7 @@ public MetricsContext createEntityIdMetricsContext(EntityId entityId) {
       return new NoopMetricsContext();
     }
     Map<String, String> tags = Collections.emptyMap();
-    if (metricsTagsEnabled && entityId != null) {
+    if (metricsTagsEnabled) {
       tags = createEntityIdMetricsTags(entityId);
     }
     return metricsCollectionService == null ? new NoopMetricsContext(tags)
```

Comment:
we can check before this `return`, if the metric does not contain `NAMESPACE` tag, let's add it.